compiler/src/dmd/e2ir.d

```diff
@@ -1725,50 +1725,10 @@ elem* toElem(Expression e, IRState *irs)
             return el_long(TYint, 0);
         }
 
```

Comment:
The above betterC check could be probably moved to the frontend. You could check `sc.func.needsCodegen`, however, that is a topic for a different PR.